File: Basics_with_Python/PythonBasicCourse/4_For Loop/10_For Loop - Exercise/07. Trekking Mania.py
n = int(input())
musala = 0
monblan = 0
kilimanjaro = 0
k2 = 0
everest = 0
total_people = 0
musala_ppl = 0
monblan_ppl = 0
kilimanjaro_ppl = 0
k2_ppl = 0
everest_ppl = 0

for number in range(1, n + 1):
    people_in_group = int(input())
    if people_in_group <= 5:
        musala_ppl += people_in_group
    elif 6 <= people_in_group <= 12:
        monblan_ppl += people_in_group
    elif 13 <= people_in_group <= 25:
        kilimanjaro_ppl += people_in_group
    elif 26 <= people_in_group <= 40:
        k2_ppl += people_in_group
    else:
        everest_ppl += people_in_group
total_people = musala_ppl + monblan_ppl + kilimanjaro_ppl + k2_ppl + everest_ppl
musala = musala_ppl / total_people * 100
monblan = monblan_ppl / total_people * 100
kilimanjaro = kilimanjaro_ppl / total_people * 100
k2 = k2_ppl / total_people * 100
everest = everest_ppl / total_people * 100

print(f"{musala:.2f}%")
print(f"{monblan:.2f}%")
print(f"{kilimanjaro:.2f}%")
print(f"{k2:.2f}%")
print(f"{everest:.2f}%")

File: Basics_with_JavaScript/910_For_Loop/10_For_Loop_Excercise/07._Trekking_Mania.js
function vladi(input) {
  let groups = Number(input[0]);
  let musalaQty = 0;
  let monblanQty = 0;
  let kilimQty = 0;
  let everestQty = 0;
  let k2Qty = 0;
  let counter = 0;

  for (let index = 1; index < input.length; index++) {
    peopleInGroup = Number(input[index]);

    if (peopleInGroup <= 5) {
      musalaQty += peopleInGroup;
      counter += peopleInGroup;
    } else if (peopleInGroup > 5 && peopleInGroup <= 12) {
      monblanQty += peopleInGroup;
      counter += peopleInGroup;
    } else if (peopleInGroup > 12 && peopleInGroup <= 25) {
      kilimQty += peopleInGroup;
      counter += peopleInGroup;
    } else if (peopleInGroup > 25 && peopleInGroup <= 40) {
      k2Qty += peopleInGroup;
      counter += peopleInGroup;
    } else if (peopleInGroup > 40) {
      everestQty += peopleInGroup;
      counter += peopleInGroup;
    }
  }
  let musala = (100 * musalaQty / counter).toFixed(2);
  let monblan = (100 * monblanQty / counter).toFixed(2);
  let kili = (100 * kilimQty / counter).toFixed(2);
  let k2 = (100 * k2Qty / counter).toFixed(2);
  let everest = (100 * everestQty / counter).toFixed(2);
  console.log(`${musala}%`);
  console.log(`${monblan}%`);
  console.log(`${kili}%`);
  console.log(`${k2}%`);
  console.log(`${everest}%`);
}

vladi(["10", "10", "5", "1", "100", "12", "26", "17", "37", "40", "78"]);
vladi(["5", "25", "41", "31", "250", "6"]);
